scripts/src/cpu.lua

```diff
@@ -3929,6 +3929,8 @@ end
 --------------------------------------------------
 
 if opt_tool(CPUS, "UPD177X") then
+	table.insert(disasm_files , MAME_DIR .. "src/devices/cpu/upd177x/upd177x.cpp")
+	table.insert(disasm_files , MAME_DIR .. "src/devices/cpu/upd177x/upd177x.h")
 	table.insert(disasm_files , MAME_DIR .. "src/devices/cpu/upd177x/upd177xd.cpp")
```

Comment:
You shouldn’t be adding the CPU core files to the disassembler files.  If unidasm depends on the CPU core files, something is very wrong.

Reply:
That was an error from my end, there is no dependency.